### ISSUE TITLE
500 Internal Server Error - Pull List Error?

### ISSUE THREAD

**Issue by kev0153** [evilhero/mylar#1080]
I got this error while alt_pull list was on.  I turned alt_pull list off and was still getting it.

29-Jun-2015 18:41:36 - DEBUG   :: WEEKLYCHECK : no Alternate name given. Aborting search.
29-Jun-2015 18:41:36 - DEBUG   :: WEEKLYCHECK : ComicName: A-Force
29-Jun-2015 18:41:36 - DEBUG   :: WEEKLYCHECK : latestdate:  15-07-01
29-Jun-2015 18:41:36 - DEBUG   :: WEEKLYCHECK : invalid date 15-07-01 appending 01 for day for continuation.
29-Jun-2015 18:41:36 - DEBUG   :: WEEKLYCHECK : Exception generated in thread WEEKLYCHECK: invalid literal for int() with base 10: '15-0'
29-Jun-2015 18:41:36 - DEBUG   :: WEEKLYCHECK : 'Traceback (most recent call last):\n  File "/home/kev/.mylar/mylar/scheduler.py", line 78, in runAction\n    self.action.run()\n  File "/home/kev/.mylar/mylar/weeklypullit.py", line 30, in run\n    mylar.weeklypull.pullit()\n  File "/home/kev/.mylar/mylar/weeklypull.py", line 179, in pullit\n    pullitcheck()\n  File "/home/kev/.mylar/mylar/weeklypull.py", line 531, in pullitcheck\n    c_date = datetime.date(int(latestdate[:4]), int(latestdate[5:7]), int(latest_day))\nValueError: invalid literal for int() with base 10: \'15-0\'\n'

**Comment by evilhero** [comment]
It looks like the A-force title has an invalid date present. Not sure if that's from comicvine or from the pulllist. ..can you check your A-force series listing and see what the date for that issue is listed as? If it's stated as '15' instead of '2015' then that would be causing the problem and would need to get fixed.  CV usually fixes these problems in a day or so, but you can also edit that date and put in the correct value so that things keep on trucking (if that's where the incorrect date is)

**Comment by kev0153** [comment]
everything looks right with the data in comicvine and in my series names.  Is there a way I can delete this series by hand so comic vine won't trip over it?

**Comment by kev0153** [comment]
I think something else is going on.  I deleted A-Force from the sqlite database and everthing has going find until it crashed again.  

0-Jun-2015 07:58:43 - DEBUG   :: WEEKLYCHECK : no Alternate name given. Aborting search.
30-Jun-2015 07:58:43 - DEBUG   :: WEEKLYCHECK : ComicName: FBP: Federal Bureau of Physics
30-Jun-2015 07:58:43 - DEBUG   :: WEEKLYCHECK : latestdate:  15-07-01
30-Jun-2015 07:58:43 - DEBUG   :: WEEKLYCHECK : invalid date 15-07-01 appending 01 for day for continuation.
30-Jun-2015 07:58:43 - DEBUG   :: WEEKLYCHECK : Exception generated in thread WEEKLYCHECK: invalid literal for int() with base 10: '15-0'
30-Jun-2015 07:58:43 - DEBUG   :: WEEKLYCHECK : 'Traceback (most recent call last):\n  File "/home/kev/.mylar/mylar/scheduler.py", line 78, in runAction\n    self.action.run()\n  File "/home/kev/.mylar/mylar/weeklypullit.py", line 30, in run\n    mylar.weeklypull.pullit()\n  File "/home/kev/.mylar/mylar/weeklypull.py", line 179, in pullit\n    pullitcheck()\n  File "/home/kev/.mylar/mylar/weeklypull.py", line 531, in pullitcheck\n    c_date = datetime.date(int(latestdate[:4]), int(latestdate[5:7]), int(latest_day))\nValueError: invalid literal for int() with base 10: \'15-0\'\n'

I've had these series forever and it hasn't given me problems.

**Comment by evilhero** [comment]
Ahh cap that's an error with the pullist from previews again :(  They changed the date from a 2015-07-01 format to 15-07-01 which is screwing up things.

I'll try and look at this as soon as I can get to a machine cause it breaks the home page :(

**Comment by evilhero** [comment]
Latest commit should fix this problem. Bastards.

If it's still erroring out on loading the homepage, browse to the /pullist and then select the 'Recreate Pullist' option so it can regenerate everything. Problem being is that the invalid issue dates have now been propigated into the issues table for the given series.

Apparently if you don't use the alt_pull method, this error wouldn't occur (Flipping back to using the txt file option and re-creating the pullist would allow the proper dates, but it would still error out on the invalid dates in the series as they've already been added)

**Comment by kev0153** [comment]
that got it thanks

**Comment by tcn33** [comment]
I'm still seeing this issue with A-Force. Newest commit, don't think I am using the alt_pull list (didn't know there was such a thing and don't see an option for it in the UI). Just throws a 500.

```
Traceback (most recent call last):
  File "/opt/Mylar/cherrypy/_cprequest.py", line 645, in respond
    response.body = self.handler()
  File "/opt/Mylar/cherrypy/lib/encoding.py", line 188, in __call__
    self.body = self.oldhandler(*args, **kwargs)
  File "/opt/Mylar/cherrypy/_cpdispatch.py", line 29, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "/opt/Mylar/mylar/webserve.py", line 192, in searchit
    searchresults, explicit = mb.findComic(name, mode, issue=issue)
  File "/opt/Mylar/mylar/mb.py", line 153, in findComic
    comicResults = searched.getElementsByTagName(type) #('volume')
AttributeError: 'NoneType' object has no attribute 'getElementsByTagName'
```

**Comment by evilhero** [comment]
That looks like you're searching for A-force and it's timing out on returning all the results.  Are you doing a search or an 'add series' off the pullist?

I noticed when I did a search, it came back with like 15000+ items or something and ended up maxing out my api hits for the 15 minute duration. I'm betting that's what's occuring here as it seems to be saying it can't find the results on the return page from the api, which would generally mean it's hit the max api hits for the 15 minute duration (400 hits ).
